File: 2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.java
class Solution {
    public String maximumOddBinaryNumber(String s) {
        int ones = 0, zeros = 0;
        for(char c : s.toCharArray()) {
            if(c == '1') ones++;
            else zeros++;
        }

        String ans = "";
        ones--;
        while(ones-- > 0) ans += '1';
        while(zeros-- > 0) ans += '0';
        ans += '1';

        return ans;
    }
}

File: 2864-maximum-odd-binary-number/2864-maximum-odd-binary-number.py
class Solution(object):
    def maximumOddBinaryNumber(self, s):
        ones, zeros = 0, 0
        for c in s:
            if c == '1':
                ones += 1
            else:
                zeros += 1
        
        ans = ''
        ones -= 1
        while ones > 0:
            ans += '1'
            ones -= 1
        while zeros > 0:
            ans += '0'
            zeros -= 1
        
        ans += '1'

        return ans
        